gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/client.py.j2

```diff
@@ -175,24 +181,42 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
             client_options = ClientOptions.from_dict(client_options)
         if client_options is None:
             client_options = ClientOptions.ClientOptions()
+        
+        # Create SSL credentials for mutual TLS if needed.
+        use_client_cert = os.getenv("GOOGLE_API_USE_CLIENT_CERTIFICATE", "false")
+        if not use_client_cert in ["true", "false"]:
+            raise MutualTLSChannelError(
+                "Unsupported GOOGLE_API_USE_CLIENT_CERTIFICATE value. Accepted values: true, false"
+            )
+
+        ssl_credentials = None
+        is_mtls = False
+        if use_client_cert == "true":
+            if client_options.client_cert_source:
+                cert, key = client_options.client_cert_source()
```

Comment:
I think I'd be happier moving the `import grpc` to here because it removes the hard dependency; if users don't want to use mtls, they're not forced to import grpc in the client, which makes it easier to rely on alternative transports.

Reply:
done